src/components/DragCaptureZone/examples/interactive.jsx

```diff
@@ -41,26 +41,21 @@ export default React.createClass({
 					overflow: 'auto',
 					width: 600,
 				}}>
-					{
-						_.chain(this.state.events)
```

Comment:
Had to remove all usage of `chain`. In most cases I just went with nesting, but for longer chains I used `lodash/fp/flow`